projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/components/linked-cases-table/linked-cases-from-table.component.spec.ts

```diff
@@ -0,0 +1,202 @@
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { ActivatedRoute } from '@angular/router';
+import { of, throwError } from 'rxjs';
+import { AbstractAppConfig } from '../../../../../../app.config';
+import { PipesModule } from '../../../../../pipes/pipes.module';
+import { SearchService } from '../../../../../services';
+import { CasesService } from '../../../../case-editor/services/cases.service';
+import { CaseLink } from '../../domain/linked-cases.model';
+import { LinkedCasesService } from '../../services';
+import { LinkedCasesFromTableComponent } from './linked-cases-from-table.component';
+
+import createSpyObj = jasmine.createSpyObj;
+
+describe('LinkCasesFromTableComponent', () => {
+  let component: LinkedCasesFromTableComponent;
+  let fixture: ComponentFixture<LinkedCasesFromTableComponent>;
+  let casesService: any;
+  let searchService: any;
+  let nativeElement: any;
+  let appConfig: any;
+  let linkedCasesService: any;
+
+  const linkedCases: CaseLink[] = [
+    {
+      caseReference: '1682374819203471',
+      reasons: [],
+      createdDateTime: '',
+      caseType: 'SSCS',
+      caseTypeDescription: 'SSCS case type',
+      caseState: 'state',
+      caseStateDescription: 'state description',
+      caseService: 'Tribunal',
+      caseName: 'SSCS 2.1',
+    },
+    {
+      caseReference: '1682897456391875',
+      reasons: [],
+      createdDateTime: '',
+      caseType: 'SSCS',
+      caseTypeDescription: 'SSCS case type',
+      caseState: 'state',
+      caseStateDescription: 'state description',
+      caseService: 'Tribunal',
+      caseName: 'SSCS 2.1',
+    },
+  ];
+
+  linkedCasesService = {
```

Comment:
@johnbenjamin-hmcts - this could be declared immediately at Line 22

Reply:
Thank you @Daniel-Lam, I have now updated the code.